src/main/java/com/google/flourbot/entity/action/sheet/SheetAppendRowAction.java

```diff
@@ -0,0 +1,49 @@
+package com.google.flourbot.entity.action.sheet;
```

Comment:
btw, It would be nice to separate these two changes, because they address completely different functionality but that's ok for now.